purpleair-aqi.js

```diff
@@ -189,32 +189,23 @@ function haversine(start, end) {
 async function getSensorData(sensorId) {
 
   const req = new Request(`${API_URL}/json?show=${sensorId}`);
-
-    const json = await req.loadJSON();
+  const json = await req.loadJSON();
 
   try {
-
-  return {
-    val: json.results[0].Stats,
-    adj1: json.results[0].pm2_5_cf_1,
-    adj2: json.results[1].pm2_5_cf_1,
-    ts: json.results[0].LastSeen,
-    hum: json.results[0].humidity,
-    loc: json.results[0].Label,
-    lat: json.results[0].Lat,
-    lon: json.results[0].Lon,
-  };
-   } catch (error) {
+    return {
+      val: json.results[0].Stats,
+      adj1: json.results[0].pm2_5_cf_1,
+      adj2: json.results[1].pm2_5_cf_1,
+      ts: json.results[0].LastSeen,
+      hum: json.results[0].humidity,
+      loc: json.results[0].Label,
+      lat: json.results[0].Lat,
+      lon: json.results[0].Lon,
+    };
+  } catch (error) {
     console.log(`Could not parse JSON: ${error}`);
-
-  return {
-    val: 666,
-  };
-
+    throw 666;
```

Comment:
This basically will squash the current error, and then re-throw it as just a `666`